conf/tachyon-env.sh.template

```diff
@@ -101,6 +101,8 @@ export TACHYON_JAVA_OPTS+="
   -Dtachyon.worker.tieredstore.level0.dirs.path=${TACHYON_RAM_FOLDER}
   -Dtachyon.worker.tieredstore.level0.dirs.quota=${TACHYON_WORKER_MEMORY_SIZE}
   -Dtachyon.underfs.address=${TACHYON_UNDERFS_ADDRESS}
+  -Dtachyon.underfs.glusterfs.volumes=${TACHYON_UNDERFS_GLUSTERFS_VOLUMES}
```

Comment:
It's useful when GlusterFS is used as under FS. Without defining those variables, the options will be configured as null and therefore ignored.

We can also create a template file for gluster only, but then we may need to synchronize the changes among the templates.